_field-types/supported-field-types/star-tree.md

```diff
@@ -36,7 +36,7 @@ The following examples show how to use a star-tree index.
 
 Define star-tree index mappings in the `composite` section in `mappings`. 
 
-The following example API request creates a corresponding star-tree index for all `request_aggs`. To compute metric aggregations for `request_size` and `latency` fields with queries on `port` and `status` fields, configure the following mappings:
+The following example API request creates a corresponding star-tree index for `request_aggs`. To compute metric aggregations for `request_size` and `latency` fields with queries on `port` and `status` fields, configure the following mappings:
```

Comment:
The following example API request creates a corresponding star-tree index named `request_aggs` for request aggregations

`request_aggs` is the star tree index name - maybe we can word it like above